RELEASE.md

```diff
@@ -0,0 +1,91 @@
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# Release Notes for 2.37.0
+
+## New Freatures and Improvements
+
+* Triton can load model instances in parallel for supporting backends. See [TRITONBACKEND_BackendAttributeSetParallelModelInstanceLoading](https://github.com/triton-inference-server/backend/tree/r23.08#tritonbackend_backendattribute) for more details. As of 23.08, only [python](https://github.com/triton-inference-server/python_backend/tree/r23.08) and [onnxruntime](https://github.com/triton-inference-server/onnxruntime_backend/tree/r23.08) backends support loading model instances in parallel.
+
+* Python backend models can capture [trace for composing child](https://github.com/triton-inference-server/server/blob/r23.08/docs/user_guide/trace.md#tracing-for-bls-models) models when executing BLS requests.
+
+* Triton OpenTelemetry Tracing exposes [resource settings](https://github.com/triton-inference-server/server/blob/r23.08/docs/user_guide/trace.md#opentelemetry-trace-apis-settings) which can be used to configure the service name and version.
+
+* Python backend supports directly [loading and serving PyTorch models](https://github.com/triton-inference-server/python_backend/tree/r23.08#pytorch-platform-experimental) with torch.compile().
+
+* Exposed [preserve_ordering](https://github.com/triton-inference-server/common/blob/r23.08/protobuf/model_config.proto#L1461-L1481) field to oldest strategy sequence batcher. The default behavior of the oldest strategy sequence batcher to preserve response order across the independent requests belonging to different sequences is changed from True to False. Note: This setting does not impact order of responses within a sequence.
+
+* Refer to the 23.08 column of the
+  [Frameworks Support Matrix](https://docs.nvidia.com/deeplearning/frameworks/support-matrix/index.html)
+  for container image versions on which the 23.08 inference server container is
+  based.
+
+## Known Issues
+
+* Triton uses OpenTelemetry CPP library version, which can cause Triton to [crash](https://github.com/triton-inference-server/server/issues/6202), when OpenTelemetry’s exporter timeouts.
+
+* When using decoupled models, there is a possibility that response order as sent from the backend may not match with the order in which these responses are received by the streaming gRPC client.
+
+* The
+  ["fastertransformer_backend"](https://github.com/triton-inference-server/fastertransformer_backend) is only officially supported for 22.12, though it can be built for Triton container versions up to 23.07.
```

Comment:
Are we keep supporting "fastertransformet_backend" ?

Reply:
No... FT cannot be built with cuda 12.2.1 which is present in 23.08... We can build FT backend till 23.07 though with the latest fix from FT team.

Reply:
interested as we have that `r23.08` branch in that repo...

Reply:
I have deleted it